dkc/core/tests/factories.py

```diff
@@ -33,6 +34,6 @@ class Meta:
     name = factory.Faker('file_name')
     description = factory.Faker('paragraph')
     blob = factory.django.FileField(data=b'fakefilebytes', filename='fake.txt')
-    creator = factory.SubFactory(UserFactory)
+    creator = factory.LazyAttribute(lambda o: o.folder.owner)
```

Comment:
```suggestion
    creator = factory.SelfAttribute('folder.owner')
```

Can this be simplified as [a `SelfAttribute`](https://factoryboy.readthedocs.io/en/stable/reference.html#factory.SelfAttribute)?